<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav)"/>
  <arg name="world_frame_name" default="$(arg uav_name)/stable_origin"/>

  <group ns="$(arg uav_name)">

    <node pkg="nodelet" type="nodelet" name="compton_plotter" args="standalone compton_camera_python/ComptonPlotter" output="screen">

      <rosparam file="$(find compton_camera_python)/config/plotter.yaml" command="load" />

      <param name="uav_name" value="$(arg uav_name)" />
      <param name="world_frame_name" value="$(arg world_frame_name)" />

      <!-- subscribed topics -->
      <remap from="~cone_in" to="compton_camera/cone" />
      <remap from="~source_in" to="/radiation/sources" />

    </node>

  </group>

</launch>
